atom/browser/native_window.h

```diff
@@ -161,6 +161,10 @@ class NativeWindow : public base::SupportsUserData,
   virtual void SetVisibleOnAllWorkspaces(bool visible) = 0;
   virtual bool IsVisibleOnAllWorkspaces() = 0;
 
+  // Vibrancy API
+  virtual void SetVibrancy(const std::string& type) = 0;
+  virtual void RemoveVibrancy() = 0;
```

Comment:
These need to be override in `atom/browser/native_window_views.h` and ``atom/browser/native_window_views.cc` as well, currently failing on Linux CI with:

```
./../atom/browser/native_window_views.cc -o obj/atom/browser/electron_lib.native_window_views.o
../../atom/browser/native_window_views.cc:1289:14: error: allocating an object of abstract class type 'atom::NativeWindowViews'
  return new NativeWindowViews(inspectable_web_contents, options, parent);
             ^
../../atom/browser/native_window.h:165:16: note: unimplemented pure virtual method 'SetVibrancy' in 'NativeWindowViews'
  virtual void SetVibrancy(const std::string& type) = 0;
               ^
../../atom/browser/native_window.h:166:16: note: unimplemented pure virtual method 'RemoveVibrancy' in 'NativeWindowViews'
  virtual void RemoveVibrancy() = 0;
```